guidelines/index.html

```diff
@@ -1103,6 +1115,8 @@
 					<p>A change of content is not always a change of context. Changes in content, such as an expanding outline, dynamic menu, or a tab control do not necessarily change the context, unless they also change one of the above (e.g., focus).</p>
 					<p>Opening a new window, moving focus to a different component, going to a new page (including anything that would look to a user as if they had moved to a new page) or significantly re-arranging the content of a page are examples of changes of context.</p>
 				</dd>
+        <dt><dfn>common input error</dfn></dt>
+        <dd>errors that have been have been documented in the understanding of the Success Criteria section or have supportive techniques or that have been reported or documented more then one time and there is a know way to identify them</dd>
```

Comment:
* Fix double "have been"
* Change "know" to "known"
* Reword as a clearer comma-separated list (too many uses of "or" without punctuation)